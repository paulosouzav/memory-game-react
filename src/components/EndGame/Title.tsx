import styled from 'styled-components';
import { COLORS, TRANSPARENT_BLACK } from '../../constants/colors';
import { mediaMax } from '../../utils/mediaQuery';

const Title = styled.h1`
  color: ${COLORS.white};

  font-size: 50px;
  text-shadow: 0 0 8px ${TRANSPARENT_BLACK.normal};
  text-align: center;
  line-height: normal;

  margin: 0 0 10px;
  padding: 0;

  ${mediaMax.sm`
    font-size: 40px;
  `}
`;

Title.displayName = 'Title';

export default Title;
