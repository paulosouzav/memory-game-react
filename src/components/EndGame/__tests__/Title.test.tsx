import React from 'react';
import { mount } from 'enzyme';
import 'jest-styled-components';
import Title from '../Title';

describe('Title', () => {
  it('should render without failing', () => {
    const wrapper = mount(<Title />);
    expect(wrapper).toMatchSnapshot();
  });
});
