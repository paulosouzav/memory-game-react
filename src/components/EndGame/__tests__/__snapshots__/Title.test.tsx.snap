// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should render without failing 1`] = `
.c0 {
  color: #ffffff;
  font-size: 50px;
  text-shadow: 0 0 8px rgba(0,0,0,0.5);
  text-align: center;
  line-height: normal;
  margin: 0 0 10px;
  padding: 0;
}

@media (max-width:576px) {
  .c0 {
    font-size: 40px;
  }
}

<Title>
  <StyledComponent
    forwardedComponent={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "sc-bdVaJa",
          "isStatic": true,
          "lastClassName": "c0",
          "rules": Array [
            "
  color: ",
            "#ffffff",
            ";

  font-size: 50px;
  text-shadow: 0 0 8px ",
            "rgba(0,0,0,0.5)",
            ";
  text-align: center;
  line-height: normal;

  margin: 0 0 10px;
  padding: 0;

  ",
            "
      @media (",
            "max",
            "-width: ",
            "576",
            "px) {
        ",
            "
    font-size: 40px;
  ",
            ";
      }
    ",
            "
",
          ],
        },
        "displayName": "Title",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "sc-bdVaJa",
        "target": "h1",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      }
    }
    forwardedRef={null}
  >
    <h1
      className="c0"
    />
  </StyledComponent>
</Title>
`;
